File: docs/2014/snippets/csharp/SQL15/integration services runtime api/application/cs/application_checksignatureonload.cs
//<snippetApplication_CheckSignatureOnLoadCS>
class ApplicationTests
{
    static void Main(string[] args)
    {
        Application app = new Application();

        Boolean sig = app.CheckSignatureOnLoad;
        Console.Writeline(sig);
    }
}
//</snippetApplication_CheckSignatureOnLoadCS>

File: docs/2014/snippets/visualbasic/SQL15/integration services runtime api/application/vb/application_checksignatureonload.vb
' <snippetApplication_CheckSignatureOnLoadVB>
Class ApplicationTests
    Sub Main(ByVal args() As String)
        Dim app As Application =  New Application() 
 
        Dim sig As Boolean =  app.CheckSignatureOnLoad 
        Console.Writeline(sig)
    End Sub
End Class
' </snippetApplication_CheckSignatureOnLoadVB>